pygmt/tests/test_velo.py

```diff
@@ -0,0 +1,40 @@
+"""
+Tests velo.
+"""
+import pandas as pd
+import pytest
+
+from pygmt import Figure
+
+
+@pytest.mark.mpl_image_compare
+def test_velo_arrow_ellipse_pandas_df():
+    """
+    Plot velocity arrow and confidence ellipse from a pandas.DataFrame
+    """
+    fig = Figure()
+    dframe = pd.DataFrame(
+        data={
+            "Long.": [0, -8, 0, -5, 5, 0],
+            "Lat.": [-8, 5, 0, -5, 0, -5],
+            "Evel": [0, 3, 4, 6, -6, 6],
+            "Nvel": [0, 3, 6, 4, 4, -4],
+            "Esig": [4, 0, 4, 6, 6, 6],
+            "Nsig": [6, 0, 6, 4, 4, 4],
+            "CorEN": [0.5, 0.5, 0.5, 0.5, -0.5, -0.5],
+            # "SITE": ["4x6", "3x3", "NaN", "6x4", "-6x4", "6x-4"],
```

Comment:
I'll try and figure out a way to add the text label in later :wink:

Reply:
Thanks @weiji14 , I didnt realise you could do that by just clicking a button! I pulled the changes from remote/usptream/master onto my local forked repository and push my branch again (After merging also the reviewed changes you did on the remote forked repository).